src/Microsoft.DocAsCode.Dfm.VscPreview/packages.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Composition" version="1.0.27" targetFramework="net452" />
+  <package id="System.Collections" version="4.0.0" targetFramework="net452" />
+  <package id="System.Collections.Immutable" version="1.1.37" targetFramework="net452" />
+  <package id="System.Diagnostics.Debug" version="4.0.0" targetFramework="net452" />
+  <package id="System.Globalization" version="4.0.0" targetFramework="net452" />
+  <package id="System.Linq" version="4.0.0" targetFramework="net452" />
```

Comment:
The 'System.Collections.Immutable' depends on it

---
In reply to: [74005536](https://github.com/dotnet/docfx/pull/583#discussion_r74005536) [](ancestors = 74005536)